internal/services/servicebus/servicebus_namespace_resource_test.go

```diff
@@ -139,6 +139,17 @@ func TestAccAzureRMServiceBusNamespace_premiumCapacity(t *testing.T) {
 	})
 }
 
+func TestAccAzureRMServiceBusNamespace_premiumMessagingPartition(t *testing.T) {
+	data := acceptance.BuildTestData(t, "azurerm_servicebus_namespace", "test")
+	r := ServiceBusNamespaceResource{}
+	data.ResourceTest(t, r, []acceptance.TestStep{
+		{
+			Config:      r.premiumMessagingPartition(data),
+			ExpectError: regexp.MustCompile("Service Bus SKU \"Premium\" only supports `premium_messaging_partitions` of 1, 2, 4"),
```

Comment:
This doesn't match the schema validation above?

Reply:
The validation error is for the premium namespaces